src/components/Container/ContainerMarginHelper.ts

```diff
@@ -0,0 +1,120 @@
+import { IContainerProps } from './Container';
+
+export type ContainerMarginType = 'xs' | 's' | 'm' | 'l' | 'xl' | 0 | 'none' | string;
+type ReturnValue =  {margin?: string | number} | undefined;
```

Comment:
Is this type compatible with props like `marginTop`? Should it be something like `{[margin: 'margin'|'marginTop'|...]: string | number}`?

Just caught my eye, I may be way off here.

Reply:
Good catch! I totally missed updating that.